jsonparse.mly

```diff
@@ -10,6 +11,69 @@ open ProcessTypes
 (* BUG: need to unescape strings
    (where they are escaped in json.ml)
 *)
+let websocket_req assoc_list =
+  let open Pervasives in
```

Comment:
It would be good to mention that you open `Pervasives` in order to overload the definition of `|>` in `Utility.ml`.